### ISSUE TITLE
Thirst is not ignoring players in creative mode.

### ISSUE THREAD

**Issue by TrademarkTM** [GamerKing195/Thirst#8]
Hello!
This is just a small bug I found while using your plugin. Basically, what the title says, the thirst is being removed while I am in creative mode.

## Information
My configuration for ignore creative is this one:
```
# 
# IGNORE_CREATIVE
# Desc: If true, anyone in creative will not be affected by Thirst.
# Type: Boolean
# Default: true
Ignore-Creative: true
```
It is basically ignoring this "true" in the config.
**Server version:** 1.11.2
**Plugin version:** v1.8.3

Ignore OP is working normally, though.

Thanks!

**Comment by cadenkriese** [comment]
I'll get to work on a fix for this asap, thank you for the well laid out report.

**Comment by cadenkriese** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [FIX - Ignore-Creative config node not working.](https://trello.com/c/BEjdotY1/36-fix-ignore-creative-config-node-not-working)

**Comment by cadenkriese** [comment]
This bug has been fixed in version 1.8.4 you can download it [here](https://www.spigotmc.org/resources/thirst.24610/update?update=164084).
